ps_socialfollow.php

```diff
@@ -34,8 +34,6 @@
 
 class Ps_Socialfollow extends Module implements WidgetInterface
 {
-    private $templateFile;
-
     const SOCIAL_NETWORKS = [
         'facebook',
```

Comment:
It looks to me as it was a bad choice to have these invariant strings in lower case. The configuration names are in upper case, and this requires to convert the case several times. I adds complexity without benefit.

Reply:
I agree 👍